### DISCUSSION TITLE
Example Django app using this repo

### DISCUSSION THREAD

**Discussion by davidshumway** [tiangolo/uwsgi-nginx-docker#218] [Questions] [answered]
Can someone point to an example Django app using this repo?
I built a Django app and followed the steps here (https://uwsgi-docs.readthedocs.io/en/latest/tutorials/Django_and_nginx.html#install-uwsgi-system-wide) to run it with uwsgi and nginx using a uwsgi .ini file.
What are the steps required use this repo (tiangolo / uwsgi-nginx-docker) to create a docker image using the Django app and run the image in a docker container? Thanks!

**Comment by karlhedins** [comment, marked as answer]
Made a small example with [uwsgi-nginx-docker](https://github.com/tiangolo/uwsgi-nginx-docker) configured with Django:
https://github.com/karlhedins/uwsgi-nginx-django-docker
